acrobat/blocks/verb-widget/verb-widget.js

```diff
@@ -46,6 +71,23 @@ const setDraggingClass = (widget, shouldToggle) => {
   shouldToggle ? widget.classList.add('dragging') : widget.classList.remove('dragging');
 };
 
+function redDir(verb) {
+  if (isOldBrowser()) {
```

Comment:
let's find a away to call `isOldBrowser()` regardless if the user is signed-in please.

Reply:
Hi @Blainegunn I've added it at the top of init function, please have a look and let me know what do you think about it
https://github.com/adobecom/dc/pull/800/commits/8e4d5ed908605f204ad4b08efba1aa9a961a1af4